### PR TITLE
Interop - Expand cluster-logging on ROSA to 4.17

### PR DESCRIPTION
**Pull request by oharan2**
Recent Job:
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/58425/rehearse-58425-periodic-ci-openshift-logging-extended-logging-tests-interop-main-cluster-logging-ocp4.17-lp-rosa-classic-cluster-logging-operator-rosa-aws/1855631560938622976

### CODE REVIEW

File: ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.17-lp-rosa-classic.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    namespace: ci
+    tag: latest
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.17"
```
Comment (anpingli) [inline ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.17-lp-rosa-classic.yaml:25]: Please use 4.16 for  tests-private-ui and tests-private:=

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-openshift-logging-extended-logging-tests-interop-main-cluster-logging-ocp4.17-lp-rosa-classic-cluster-logging-operator-rosa-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-openshift-logging-extended-logging-tests-interop-main-cluster-logging-ocp4.17-lp-rosa-classic-cluster-logging-operator-rosa-aws

**Pull request merged**